ocs_ci/ocs/elasticsearch.py

```diff
@@ -93,6 +142,27 @@ def _deploy_eck(self):
             except IndexError:
                 log.info("ECK operator pod not ready yet")
 
+    def _deploy_dump_client(self):
```

Comment:
it may be a good idea to rename this function name for better readability.

Reply:
renamed.